addons/web/static/src/js/views/basic/basic_model.js

```diff
@@ -2438,7 +2438,7 @@ var BasicModel = AbstractModel.extend({
 
                     this.do_warn(
                         _.str.sprintf(
-                            _t(`'%s' is unsynchronized with ''.`),
+                            _t(`'%s' is unsynchronized with '%s'.`),
```

Comment:
In stable don't forget to add it in the pot file

Reply:
I already did that in the commit that introduced the first placeholder: https://github.com/odoo/odoo/blob/0078d8f633b2d3ba312717f55e6d9c3a18b68026/addons/web/i18n/web.pot#L93

Don't know exactly how I managed to do one correctly but not the other since I thought it was a copy/paste, I guess I had a rogue selection on that placeholder while greping and removed it by accident.